packages/scanner/package.json

```diff
@@ -77,6 +77,7 @@
     "minimatch": "^3.0.4",
     "octokat": "^0.10.0",
     "openapi-diff": "^0.23.5",
+    "ora": "~5",
```

Comment:
Prior to this change, we picked up `ora` because it's a dependency of `inquirer`. We're currently using v8.2.0 of `inquirer`, which is locked to v5.4.1 of `ora`.